parallel-install/pkg/installation/installation.go

```diff
@@ -51,13 +50,17 @@ func NewInstallation(prerequisites [][]string, componentsYaml string, overridesY
 	}, nil
 }
 
+func (i *Installation) SetupLogger(log func(string, ...interface{})) {
```

Comment:
This function should be defined as top-level public function in the config package.